db/migrate/20200729191001_remove_has_brochure_from_passengers.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveHasBrochureFromPassengers < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :passengers, :has_brochure, :boolean
```

Comment:
Did you actually run this migration? I don't see a schema change.

Reply:
I ran it and it said it successfully migrated, so I'm not sure why it wouldn't change.

Reply:
https://github.com/umts/st-pax/pull/228/files#diff-1acd2e7e27a227829d5d14a91c863bb6L52

Reply:
My mistake 👯‍♂️